src/amo/components/Page/index.js

```diff
@@ -31,10 +36,28 @@ type InternalProps = {|
 export const PageBase = ({
   children,
   clientApp,
+  errorHandler,
   isHomePage = false,
   location,
   showWrongPlatformWarning = true,
 }: InternalProps) => {
+  let errorContent;
+  if (errorHandler && errorHandler.hasError()) {
+    log.info(`Captured API Error: ${errorHandler.capturedError.messages}`);
```

Comment:
This is not correct, it should be `.warn()` here

Reply:
I was intending to downgade it from `warn` to `info` because it's not really something we need to worry about in most cases. It will happen anytime an incorrect slug is used in a URL, for example. Why do you feel `debug` is better than `info`?

Reply:
`info` gets logged all the time. If that's what we want, then it should be a warning, not an info message. Both are retained in Stackdriver but let's use correct log levels.

If we don't care about this kind of messages, `debug` should be used so that those messages don't go to Stackdriver.

Reply:
I was going to go with `debug`, but then I thought that we might want to see these messages in Stackdriver in some cases, which is likely why they were being logged in the first place. But it's fair to say they're not really `info` messages. We could just switch back to `warn`, or another idea I had was to only log if the status code isn't one of the ones being caught (i.e., not 401, 403, 404 or 451). That would allow us to log any unexpected errors, and simply ignore those that we do expect.

@willdurand WDYT?

Reply:
Yep, sounds good